### PR TITLE
Feature 285 scripts for metatavu admins

### LINKED ISSUE #285: Allow adding, updating, and deleting scripts only for Metatavu admins

### PR DESCRIPTION
**Pull request by platonovsimeon**
Resolves #285

### CODE REVIEW

File: src/test/java/fi/metatavu/metaform/server/test/functional/tests/ScriptsTestsIT.kt [hunk 1]
```diff
@@ -212,8 +216,9 @@ class ScriptsTestsIT {
       type = ScriptType.EXPORT_XSLX
     )
 
-    builder.systemAdmin.scripts.create(script)
+    builder.metatavuAdmin.scripts.create(script)
 
+    builder.assertApiCallFailStatus(200) { builder.systemAdmin.scripts.list() }
```
Comment (villejuutila) [inline src/test/java/fi/metatavu/metaform/server/test/functional/tests/ScriptsTestsIT.kt:221]: How can request fail with status code 200?

File: src/test/java/fi/metatavu/metaform/server/test/functional/tests/ScriptsTestsIT.kt [hunk 2]
```diff
@@ -190,8 +193,9 @@ class ScriptsTestsIT {
       type = ScriptType.EXPORT_XSLX
     )
 
-    val createdScript = builder.systemAdmin.scripts.create(script)
+    val createdScript = builder.metatavuAdmin.scripts.create(script)
 
+    builder.assertApiCallFailStatus(200) { builder.systemAdmin.scripts.find(createdScript.id!!) }
```
Comment (villejuutila) [inline src/test/java/fi/metatavu/metaform/server/test/functional/tests/ScriptsTestsIT.kt:198]: How can request fail with status code 200?

File: src/test/java/fi/metatavu/metaform/server/test/functional/tests/ScriptsTestsIT.kt [hunk 3]
```diff
@@ -238,22 +243,22 @@ class ScriptsTestsIT {
 
     assertEquals(createdScript.id, createdMetaform.scripts!![0])
 
-    val createdScript2 = builder.systemAdmin.scripts.create(script)
-    val updatedMetaform1 = builder.systemAdmin.metaforms.updateMetaform(createdMetaform.id!!, createdMetaform.copy(
+    val createdScript2 = builder.metatavuAdmin.scripts.create(script)
```
Comment (villejuutila) [inline src/test/java/fi/metatavu/metaform/server/test/functional/tests/ScriptsTestsIT.kt:246]: Is it intentional that all scripts are only tested for metatavu admins?

**Review by villejuutila — CHANGES_REQUESTED** [review]

File: src/test/java/fi/metatavu/metaform/server/test/functional/tests/ScriptsTestsIT.kt (hunk 3 above)
Comment (platonovsimeon) [reply src/test/java/fi/metatavu/metaform/server/test/functional/tests/ScriptsTestsIT.kt:246]: Yes, creating, updating, and deleting is allowed only for Metatavu admins

**Pull request merged**